From 2e9f70e172d842322b5bd48d21963e4ed3da8c15 Mon Sep 17 00:00:00 2001
From: Andrew Bayer <andrew@cloudera.com>
Date: Thu, 22 Mar 2012 12:57:36 -0700
Subject: [PATCH 007/154] CLOUDERA-BUILD. Bumping Surefire version to 2.12. Helps prevent issues
 when running Sonar analysis, provides better logging of unit test issues.

---
 pom.xml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/pom.xml b/pom.xml
index ba95fee..742b8ed 100644
--- a/pom.xml
+++ b/pom.xml
@@ -330,7 +330,7 @@
         </plugin>
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
-          <version>2.10</version>
+          <version>2.12</version>
           <configuration>
             <forkedProcessTimeoutInSeconds>900</forkedProcessTimeoutInSeconds>
             <argLine>-enableassertions -Xmx1400m -Djava.security.egd=file:/dev/./urandom</argLine>
-- 
1.7.0.4

